__tests__/routes.test.ts

```diff
@@ -32,3 +32,82 @@ test('Get a random quote', async () => {
 	expect(response.body[0]).toHaveProperty('character');
 	expect(response.body[0]).toHaveProperty('anime');
 });
+
+test('Get a quotes from character', async () => {
+	const response = await request(server)
+		.get('/quotes/character?name=Father%20Willibald')
+		.expect('Content-type', /json/)
+		.expect(200);
+	expect(response.body).toBeDefined();
+	expect(Array.isArray(response.body)).toBeTruthy();
+	expect(response.body.length).toBeGreaterThanOrEqual(1)
+	expect(response.body[0]).toHaveProperty('text');
+	expect(response.body[0]).toHaveProperty('character');
+	expect(response.body[0]).toHaveProperty('anime');
+	expect(response.body[0].character).toBe('Father Willibald');
```

Comment:
Can you add an assertion for checking the length of the returned body to the default number of quotes for this endpoint? The same goes for the other endpoint tests that have a default limitation on returned items.